src/app/register/register.component.ts

```diff
@@ -18,37 +20,27 @@ export class RegisterComponent implements OnInit {
     header: any;
     errorMessage: string;
     disableButton = true;
-    isFocused = false;
     authMessages: any;
+    hide = true;
+    heading: any;
 
     constructor(private _regService: RegService, private _router: Router, fb: FormBuilder, private _langService: LanguageService) {
         this.regForm = fb.group({
             fname : [null, Validators.compose([Validators.required])],
             lname : [null, Validators.compose([Validators.required])],
```

Comment:
The validator for lname has to be removed as the placeholder text says "A last name if you have"

Reply:
Done.